src/components/common/SelectInput.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { array, string, object } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { parseInputErrors } from '../../utils/helpers';
+
+const SelectInput = ({
+  input,
+  defaultLabel,
+  label,
+  values,
+  meta: { touched, error }
+}) => (
+  <div>
+    {label && <label>{label}</label>}
+    <div className="options-input">
+      <select className="fields" {...input} {...{ label }}>
+        <option value="" disabled="true">{defaultLabel}</option>
+        { values.map((item, index) => (<option key={index}> {item} </option>))}
```

Comment:
Leave a space before last `}`

Reply:
Actually it should be the other way around, there should be no space after the frist `{`